Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -685,8 +690,58 @@ def __init__(
                 proxy=oauth_params.get("proxy", False),
                 headers=oauth_params.get("headers", ""),
             )
+            if self.use_jwt:
+                self.generate_jwt: Dict[Any, Any] = {"jwt": jwt_params, "oauth":oauth_params}
+                self.jwt = self.create_jwt()
         else:
             self._auth = (self._username, self._password)
+            
+
+    def check_private_key(self, private_key) -> str:
+        # Define the start and end markers
+        start_marker = "-----BEGIN PRIVATE KEY-----"
+        end_marker = "-----END PRIVATE KEY-----"
+        if isinstance(private_key, dict):
+            private_key = private_key.get('password')
+        
+        if not private_key.startswith(start_marker) or not private_key.endswith(end_marker):
+            raise ValueError("Invalid private key format")
+        # Remove the markers and replace whitespaces with '\n'
+        key_content = private_key.replace(start_marker, "").replace(end_marker, "")\
```

Comment:
Why are we removing the markers? The JWT encoder does not support it as is? Do we have other integrations? 100% we have OOTB solutions for that.